compiler/javac/src/org/jetbrains/kotlin/wrappers/symbols/JavacReferenceAnnotationArgument.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jetbrains.kotlin.wrappers.symbols
+
+import org.jetbrains.kotlin.javac.Javac
+import org.jetbrains.kotlin.load.java.structure.JavaEnumValueAnnotationArgument
+import org.jetbrains.kotlin.name.FqName
+import javax.lang.model.element.VariableElement
+
+class JavacReferenceAnnotationArgument(val element: VariableElement,
+                                       javac: Javac) : JavacAnnotationArgument(FqName(element.simpleName.toString()), javac),
+        JavaEnumValueAnnotationArgument {
+
+    override fun resolve() = JavacField(element, javac)
```

Comment:
There is no mistake here. Maybe there is a mistake in the NetBeans plugin itself.